fastparquet/api.py

```diff
@@ -300,6 +333,10 @@ def sorted_partitioned_columns(pf):
 
 
 def filter_out_cats(rg, filters):
```

Comment:
It's not clear to me what these names mean.  We should either expand them a bit or else make them private.